internal/gensupport/error.go

```diff
@@ -11,10 +11,11 @@ import (
 	"google.golang.org/api/googleapi"
 )
 
-// WrapError creates an apierror.APIError from err (one that
-// does not wrap err), wraps it in err, and returns err. If
-// err is not a googleapi.Error (or a gRPC Status), it returns
-// err without modification.
+// WrapError creates an [apierror.APIError] from err (one that does not wrap
```

Comment:
nit: I think I would drop the part in () here. If makes it harder to read. I think this is implied by explicitly passing false to the func.

Reply:
Fixed, thanks!